lib/Edited/prototype/listen.js

```diff
@@ -1,18 +1,42 @@
 'use strict'
-var on
-var TEXT_ALTERING_EVENT_TYPES
-
-on = require('add-event-handler')
-TEXT_ALTERING_EVENT_TYPES = require('../../text-altering-event-types')
+var on = require('add-event-handler')
+var Combokeys = require('combokeys')
 
 module.exports = function () {
   var self = this
 
-  self.listener = function (e) {
-    self.discern(e)
+  // a combokeys instance for character addition
+  self.anyCharCombokeys = new Combokeys(self.element)
+  // `any-character` is a combokeys special argument that
+  // calls back on any character
+  self.anyCharCombokeys.bind('any-character', function () {
```

Comment:
[`any-character` calls back only on characters inserted](https://github.com/PolicyStat/combokeys/blob/master/README.md#getting-started).

I [trust `keypress`](https://github.com/PolicyStat/combokeys/blob/40eed71c5613baff80241c122a90910862677fa9/Combokeys/prototype/getMatches.js#L25) for [mdn](https://developer.mozilla.org/en-US/docs/Web/Events/keypress) and [quirksmode](http://www.quirksmode.org/dom/events/keys.html).